contact.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Product Landing Page</title>
+</head>
+
+<body>
+    <header id="header">
+        <nav id="nav-bar">
+            <ul id="navigation-bar"></ul>
+            <div>
+                <img id="header-img" src="gnysw2objzekbagrqiax.png" alt="Microverse">
+            </div>
+			<form action="index.html">
+			<ul>
+            <li><button type="button" class="nav-link">Home</button></li>
+			</ul>
+			</form>
+			<form action="contact.html">
+			<ul>
+            <li><button type="button" class="nav-link">Contact Us</button></li>
+			</ul>
```

Comment:
* Kindly explain why this change was requested this would enable the student to know why the change is important and also learn from it 👍 
* For example This is a well-explained request: `Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible`

Reply:
I have modified my comment as per the suggestions.